ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -123,7 +123,7 @@ const nonEmpty = (item: string): boolean => item && !!item.trim();
 
 const EraRequiredSwitch = (props: {tierConfig: InstitutionTierConfig, onChange: (boolean) => void}) => {
   const {tierConfig, onChange} = props;
-  const {config: {enableRasLoginGovLinking}} = useStore(serverConfigStore);
+  const {config: {enableRasLoginGovLinking, enforceRasLoginGovLinking}} = useStore(serverConfigStore);
```

Comment:
this is not used

Reply:
removed the unused one. Initially I want the toggle being controlled by enforce and enable. That means in the future, if RAS is not enforced in one env, we will not allow institution switch this toggle. But now I feel that we should allow admin switch the toggle regardless RAS enforcement. 
That said, if RAS is not enabled, eRA can be required or not depends on institution needs. 
In in real world, that is only for preprod in which eRA and RAS both should be disabled. Showing this is fine I guess.